src/components/form/LoginForm.js

```diff
@@ -217,8 +217,6 @@ export default function LoginForm({ containerStyle, stackStyle }) {
   const phoneNoError =
     phoneNoRef.current?.value.length !== 0 && !!errors?.phoneNo;
 
-  const showErrorMessage = !!errors.phoneNo || !!errors.password;
-
   const onSubmit = useCallback(
```

Comment:
dont need callback for submitting form as it's used here and the dependencies change quite frequently

Reply:
some functions here is the same

Reply:
useCallback is neccessary and valuable when pass the function down. Some optimization wont effective should be simplified